daemon/execdriver/lxc/driver.go

```diff
@@ -87,6 +87,12 @@ func (d *driver) Run(c *execdriver.Command, pipes *execdriver.Pipes, startCallba
 		"-n", c.ID,
 		"-f", configPath,
 	}
+
+	// From lxc>=1.1 the default behavior is to daemonize containers after start
+	if d.version() >= "1.1" {
```

Comment:
Also, isn't a string comparison here kind of dangerous since we really need
a numeric comparison?